debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -1094,6 +1094,36 @@ public void shouldReceiveHStoreTypeWithNullValuesAsJsonString() throws Exception
         assertInsert(INSERT_HSTORE_TYPE_WITH_NULL_VALUES_STMT, 1, schemaAndValueFieldForJsonEncodedHStoreTypeWithNullValues());
     }
 
+    @Test
+    @FixFor("DBZ-1814")
+    public void shouldReceiveByteaRawString() throws Exception {
```

Comment:
This looks good. When pulling this up, enabling the option also for other binary types (BLOB etc.), they should get some testing, too.

Reply:
DONE (I only additionally created tests in the MySQL connector for the other binary types)